tests/modules/services/way-displays/configuration.nix

```diff
@@ -0,0 +1,43 @@
+{ config, ... }:
+{
+  config = {
+    services.way-displays = {
+      enable = true;
+      package = config.lib.test.mkStubPackage { };
+      settings = {
+        CALLBACK_CMD = "notify-send \"way-displays \${CALLBACK_LEVEL}\" \"\${CALLBACK_MSG}\"";
```

Comment:
Here you show it being passed as a settings config, which makes more sense since it would be opt in

Reply:
The reason I included this is that this is the default setting, but it is not guaranteed to work out-of-the-box if notify-send isn't in the path. I guess we could patch the package in nixpkgs too, but this seemed like an easier solution (albeit with the alternative maintenance load of having to keep the default up to date).